t/ui/18-tests-details.t

```diff
@@ -158,6 +158,8 @@ my @worker_text = $get->tx->res->dom->find('#assigned-worker')->map('all_text')-
 like($worker_text[0], qr/[ \n]*Assigned worker:[ \n]*localhost:1[ \n]*/, 'worker displayed when job running');
 my @worker_href = $get->tx->res->dom->find('#assigned-worker a')->map(attr => 'href')->each;
 is($worker_href[0], '/admin/workers/1', 'link to worker correct');
+my @group_description_text = $get->tx->res->dom->find('#group-description')->map('all_text')->each;
```

Comment:
I am confused, what are you referring to with "group description text"?

Reply:
I tried to do the same way as test for assigned-worker, it uses `woker_text` variable. Test group is referenced as `$job->group` in code and description is `$job->group->description`. Thus variable is group_description + _text .